mne/preprocessing/maxwell.py

```diff
@@ -47,6 +52,9 @@ def maxwell_filter(raw, origin=(0, 0, 40), int_order=8, ext_order=3,
     st_corr : float
         Correlation limit between inner and outer subspaces used to reject
         ovwrlapping intersecting inner/outer signals during spatiotemporal SSS.
+    fine_cal_fname : str | None
```

Comment:
Let's just call this `fine_cal` and say it should be a filename in the description. Not worth the cost in characters to have `_fname` here IMO.